src/content/docs/troubleshooting.mdx

```diff
@@ -147,7 +147,7 @@ version and build numbers in order to target patches at specific releases. You
 can fix this problem by either setting `manageAppVersionAndBuildNumber` to false
 or removing the value from your export options .plist file.
 
-## I see a `Your app contains asset changes` warning when running `shorebird patch` (#asset-changes)
+## I see a `Your app contains asset changes` warning when running `shorebird patch`
```

Comment:
Does the #asset-changes thing do anything?  Was that the perma-link name?

Reply:
I think this worked before the migration to astro to make the link to this page be troubleshooting#asset-changes. I couldn't find any links in our CLI or our docs site pointing to this header.